ui/src/services/swap.js

```diff
@@ -12,47 +15,112 @@ const makeMatchingInvitation = async ({
   sellingPrice,
   boughtFor,
   walletP,
-  BuyerExclusiveInvitation,
-  publicFacetSwap,
+  publicFacetSimpleExchange,
   cardOffer,
   setLoading,
   onClose,
+  dispatch,
 }) => {
-  const result = await E(publicFacetSwap).makeMatchingInvitation({
+  console.log('cardPursePetname:', cardPurse.pursePetname);
+  console.log('cardbrand:', cardPurse.brand);
+  console.log(tokenPurses, 'tokenPurse');
+  console.log(cardDetail, 'cardDetail');
```

Comment:
Are these logs required somewhere? If not, remove please

Reply:
No . Will remove